build.sh

```diff
@@ -178,6 +184,49 @@ setup_build_environment() {
   BINDIR="${BINROOT}/${FULL_VARIANT}/${OUTDIR}"
 }
 
+#-----------------------------------------------------------------------------
+# Function: prepare_coverage_capture
+# Run lcov preparations before testing for coverage
+#-----------------------------------------------------------------------------
+prepare_coverage_capture() {
+  [[ -n $GITHUB_ACTIONS ]] && echo "::group::Code Coverage Preparation"
```

Comment:
What is the purpose of these statements? Just log messages?

Reply:
Yes, this is for a nice grouping (folding) of the log prints, to reduce noise